tabular/src/autogluon/tabular/predictor/predictor.py

```diff
@@ -4138,6 +4138,32 @@ def _assert_is_fit(self, message_suffix: str = None):
                 error_message = f"{error_message} `.{message_suffix}`."
             raise AssertionError(error_message)
 
+    def get_conformal_predictions(self, test_data, model: str) -> (List[np.ndarray], List[np.ndarray], List[np.ndarray]):
+        self._assert_is_fit("get_conformal_predictions")
+        if self.problem_type != REGRESSION:
```

Comment:
I assume `self.problem_type` can be `REGRESSION` or `CLASSIFICATION`? Do you have a separate case for binary classification, or is this part of `CLASSIFICATION`?